client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-build/windows-frameworks/windows-frameworks.component.ts

```diff
@@ -93,32 +93,35 @@ export class WindowsFramworksComponent implements OnInit, OnDestroy {
     private _cacheService: CacheService,
     private _translateService: TranslateService
   ) {
-    this.wizard.buildSettings
-      .get('pythonSettings')
-      .get('framework')
-      .valueChanges.takeUntil(this._ngUnsubscribe$)
-      .subscribe(val => {
-        this.setupValidators(val);
-      });
+    this.setupValidators();
   }
 
   get getFramework() {
     return this.selectedPythonFramework;
   }
-  private setupValidators(val) {
+  private setupValidators() {
     this.requiredValidator = new RequiredValidator(this._translateService, false);
-    if (this.wizard.wizardValues.buildSettings.applicationFramework === WebAppFramework.Python && val === PythonFrameworkType.Django) {
-      this.wizard.buildSettings
-        .get('pythonSettings')
-        .get('djangoSettingsModule')
-        .setValidators([this.requiredValidator.validate.bind(this.requiredValidator)]);
-      this.wizard.buildSettings
-        .get('pythonSettings')
-        .get('djangoSettingsModule')
-        .updateValueAndValidity();
-    } else {
-      this.removeValidators();
-    }
+    this.wizard.buildSettings.get('applicationFramework').setValidators([this.requiredValidator.validate.bind(this.requiredValidator)]);
+    this.wizard.buildSettings.get('applicationFramework').updateValueAndValidity();
+
+    this.wizard.buildSettings
+      .get('pythonSettings')
+      .get('framework')
+      .valueChanges.takeUntil(this._ngUnsubscribe$)
+      .subscribe(val => {
+        if (this.wizard.wizardValues.buildSettings.applicationFramework === WebAppFramework.Python && val === PythonFrameworkType.Django) {
+          this.wizard.buildSettings
```

Comment:
Pull latest code, and accommodate for Flask project name validators

Reply:
@issacnitin, I can't see that change in vinodkumar3 fork's dev branch, I think you did not merge it in 
this fork. Can you raise a PR and merge in this fork's dev branch ?

Reply:
I think that'll complicate the histories of both repos, since I have to cherry-pick, as the branch I merged is ahead of vinod's repo's dev branch, if I raise a PR, there'll be lot of changes.. and then we're raising a PR to the Azure's repo where the file is already changed by a different commit.. Maybe we can update this repo's dev with Azure/azure-functions-ux's dev

Reply:
@issacnitin did that now, can you check if your changes are properly accommodated.

Reply:
Looks good, thanks!